src/live-property.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.*
+ */
+
+import {Device, Property} from 'gateway-addon';
+import {WledDescription} from './wled';
+import {WledDevice} from './wled-device';
+
+export class LiveProperty extends Property<boolean> {
+
+  // eslint-disable-next-line no-unused-vars
+  constructor(device: Device) {
```

Comment:
If you use `WledDevice` here, you won't need the casts below.

Reply:
To me this seems not to work, because `getDevice` signature tells it will always return type `Device`

Reply:
That correct but you use something like `wledDevice: WledDevice` to access the device via `this.wledDevice`.

Reply:
https://github.com/tim-hellhake/wled-adapter/commit/7a0ece16825823361c443bb9b9aff721da40c70e

That's what I meant :smile: